app/gui2/src/stores/suggestionDatabase/index.ts

```diff
@@ -80,5 +82,6 @@ export const useSuggestionDbStore = defineStore('suggestionDatabase', () => {
   ])
 
   const synchronizer = new Synchronizer(entries, groups)
+  window.__entries = entries
```

Comment:
Why is it needed?

Reply:
was used for debugging, will remove, thanks